config/motoros2_config.yaml

```diff
@@ -284,3 +284,35 @@ publisher_qos:
 #
 # DEFAULT: false
 #ignore_missing_calib_data: false
+
+#-----------------------------------------------------------------------------
+# Should MotoROS2 broadcast debug messages?
+#
+# If enabled, this will broadcast log messages on the network on port UDP 21789.
+# the user can use the debug script provided in the distribution of MotoROS2 to
```

Comment:
```suggestion
# The user can use the debug script provided in the distribution of MotoROS2 to
```

unfortunately, we actually *don't* include the logger script in release `.zip`s.